source/features/edit-readme.tsx

```diff
@@ -10,7 +10,7 @@ import {isPermalink} from '../github-helpers';
 import getDefaultBranch from '../github-helpers/get-default-branch';
 
 async function init(): Promise<void | false> {
-	const readmeHeader = await elementReady('#readme .Box-header h2');
+	const readmeHeader = await elementReady('#readme .Box-title');
```

Comment:
What about GHE?

```suggestion
	const readmeHeader = await elementReady([
	'#readme .Box-title',
	'#readme .Box-header h2' // GHE #4208 
	].join());
```

Reply:
This selector works on both old and new headers.